Library/TestHelpers/TestCase.swift

```diff
@@ -25,6 +25,12 @@ internal class TestCase: FBSnapshotTestCase {
 
   override func setUp() {
     super.setUp()
+
+    let preferredDeviceName = "iPhone 8"
+    if !UIDevice.current.name.contains(preferredDeviceName) {
```

Comment:
Should we be getting this from `AppEnvironment.current` so that we can potentially inject our own value down the road (and maybe for consistency)?

Reply:
I don't think this is really for `AppEnvironment`, that's really only for dependency injection and I don't know that we'd want to inject stuff in unit tests _for_ unit tests 🤔 I think this is like a safety thing much like we have in our swiftlint.yml to fail if we leave `self.recordMode = true` in the test.